app/helpers/application_helper.rb

```diff
@@ -2,7 +2,7 @@ module ApplicationHelper
   # rubocop:disable Performance/OpenStruct
   DELETED_USER = OpenStruct.new(
     id: nil,
-    darker_color: HexComparer.new(bg: "#19063A", text: "#dce9f3").brightness,
+    darker_color: Hex::Comparer.new(["#19063A", "#dce9f3"]).brightness,
```

Comment:
To be honest...I'm not sure how this was working before 🤔 . I updated the syntax.

Reply:
....I just re-read `HexComparer` twice to figure out how this would have functioned with named arguments and....I am guessing it just didn't work 😓

Reply:
maybe we can move those two colors as a frozen array